packages/scaffold/src/partials/w3m-account-tokens-widget/index.ts

```diff
@@ -29,10 +29,6 @@ export class W3mAccountTokensWidget extends LitElement {
     this.unsubscribe.forEach(unsubscribe => unsubscribe())
   }
 
-  public override firstUpdated() {
```

Comment:
After completing convert. We should watch for the changes. I thought we could just refetch the balance after convert is done, but we are not sure if the balances will be updated right after the convert.

So instead, I wanted to fetch balances periodically and stop fetching when the screen is unmounted in `w3m-account-wallet-features-widget`